content/blog/2021-03-26-cdn-demo.md

```diff
@@ -0,0 +1,254 @@
+4. Admin creates the storage to the edge server
+    - Clients connected to that edge server retrieve the data locally
+
+Final scenario:
+- Both origin and edge servers contain the files within the folder «fs_storage/…»
+
+
+## Initial configuration
+
+Zenoh and the related projects in its ecosystem are open-source projects available on github. For this demo we use the following projects
+
+{{<table "table table-striped table-bordered">}}
+| Projects                   |  Overview                                        |
+|:---------------------------|:-------------------------------------------------|
+| **Zenoh**                  | Zero overhead pub/sub, store/query and compute protocol [[source code]](github.com/eclipse-zenoh/zenoh)  |
+| **Zenoh-backend-fielsystem** | Backend and Storages for zenoh using the file system [[source code]](github.com/eclipse-zenoh/zenoh-backend-filesystem) |
+| **Zenoh-python**            | Python client API for zenoh [[source code]](github.com/eclipse-zenoh/zenoh-python) |
+{{</table>}}
+
+
+and we initialize each node with the minimal set of tools to be used:
+
+1.	Install Cargo and Rust (nightly-build)
+2.	Build the “Zenoh” project and the “Zenoh-backend-filesystem” project
+    > **Warning:** the 2 project must be built with the same Rustc version to enable compatilities between them.
+3.	Compile both projects and setup Zenoh with the zbackend_fs library in ~/.zenoh/lib ([see the doc]( https://github.com/eclipse-zenoh/zenoh-backend-filesystem/blob/master/README.md#file-system-backend-for-eclipse-zenoh))
+4.	Install Python3 for the client calls and setup the Zenoh-python project ([see the doc](https://github.com/eclipse-zenoh/zenoh-python/blob/master/README.md#eclipse-zenoh-python-api))
+
+
+## Run the Demo
+
+**Execute Zenoh router for each server**
+    
+    # Set the router options:
+
+    $ address="tcp/<ip_node>"
+    $ port="7447"
+    $ api_port="8001"
+    $ ZENOH_OPTIONS=" -l $address:$port
+                      --rest-http-port$api_port
+	                  {-e $address:$connecting_port}"
+
+    $ ./target/release/zenohd $ZENOH_OPTIONS
+
+In particular, the option -e of Zenoh is used to connect the routers together. This option is required for edge servers that want to connect to the origin server or P2P/mesh configuration and is not needed on the origin server.
+
+
+**Creates the backend on all Zenoh routers and the storage only on the origin server** 
+
+In fact the edge servers are activated to dispatch the content only if local users request it, thus, initially, the edge servers do not have any storages which are created on demand in relation to the traffic on the region.
+[Check how to create backend and storage on Zenoh](https://github.com/eclipse-zenoh/zenoh-backend-filesystem#file-system-backend-for-eclipse-zenoh)
+
+After adding the backend on the origin server and on the edge server, Zenoh routers will consider *$ZBACKEND_FS_ROOT* directory as the root folder where to store future data.
```

Comment:
You should probably introduce the `$ZBACKEND_FS_ROOT` env variable and advise to set it before starting the router (or explaining the behaviour if not set).